core/injector/block_producing_node_injector.hpp

```diff
@@ -62,6 +62,10 @@ namespace kagome::injector {
             }),
         di::bind<runtime::Grandpa>.template to<runtime::dummy::GrandpaDummy>()
             [boost::di::override],
+        di::bind<crypto::CryptoStore>.template to(
+            [keystore_path](const auto &injector) {
+              return get_crypto_store(keystore_path, injector);
+            })[boost::di::override],
```

Comment:
Does default injection exists? Seems syncing node wouldn't to instantiate crypto storage for himself. Ensure, pls

Reply:
yes, it resides in base injector